src/main/scala/cognite/spark/v1/CdpConnector.scala

```diff
@@ -47,7 +47,8 @@ object CdpConnector {
       Constants.SparkDatasourceVersion,
```

Comment:
It would be useful to allow setting this as well. Jetfire would like to override it, I think.

Reply:
Application name is required, while the client tag is optional. Would it not make more sense that we use the application name to identify the spark data source version, and use the client tag to identify e.g. jetfire? Otherwise, we would have to start requiring the user to specify the application name, which is a breaking change.

Reply:
@sebbert Client tag is meant to be used for grouping related requests, in our case it could be all requests related to one Spark job.

We can keep this (the Spark data source version) as the default application name, and allow the user to override it. Since it's optional, it wouldn't be a breaking change.

Reply:
Ah right, that makes more sense. I'll update the PR 👍

Reply:
Nevermind, @mathialo already did :)